src/FIO/TransactionBuilder.h

```diff
@@ -31,35 +31,35 @@ class TransactionBuilder {
 public:
     static constexpr auto ApiAccountAddress = "fio.address";
     static constexpr auto ApiAccountToken = "fio.token";
+    static constexpr auto ApiAccountPayRequest = "fio.reqobt";
```

Comment:
I noticed this actually is an eos smart contract, same for `fio.token`, `fio.address`, maybe `ContractPayRequest` is more meaningful

Reply:
done